### ISSUE TITLE
Wrong highlight order in the TOC (for wide screen)

### ISSUE THREAD

**Issue by aborruso** [squidfunk/mkdocs-material#3578]
### Contribution guidelines

- [X] I've read the [contribution guidelines](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md) and wholeheartedly agree

### I've found a bug and checked that ...

- [X] ... the problem doesn't occur with the `mkdocs` or `readthedocs` themes
- [X] ... the problem persists when all overrides are removed, i.e. `custom_dir`, `extra_javascript` and `extra_css`
- [X] ... the documentation does not mention anything about my problem
- [X] ... there are no open or closed issues that are related to my problem

### Description

In my 2560 x 1028, when I click on a menu in the right TOC, it remains with the active CSS style the menu previous to the one I clicked on.
If I resize my windows, all works properly.

In this GIF you can see it in action

![material](https://user-images.githubusercontent.com/30607/153776181-578eefef-9b1d-4837-b4cd-bf9a3adc0c0f.gif)

### Expected behaviour

The active CSS style should be applied to the menu item I have clicked on.

### Actual behaviour

The active CSS style is applied to the menu item previous to the one I clicked on.

### Steps to reproduce

To use a resolution as the one I use and to browse https://ondata.github.io/guidaVisiData/ricette/

### Package versions

- Python: `python --version` 

Python 3.9.2

- MkDocs: `mkdocs --version`

mkdocs, version 1.2.3

- Material: `pip show mkdocs-material | grep -E ^Version`

Version: 8.1.3

### Configuration

```yaml
site_name: Guida VisiData
site_url: https://ondata.github.io/guidaVisiData/
```

### System information

- Operating system:  debian 11
- Browser: Chrome OS

**Comment by squidfunk** [comment]
Thanks for reporting. Indeed this is an issue and probably related to the upsizing of the overall content.

**Comment by squidfunk** [comment]
Fixed in a7aa55258.

**Comment by aborruso** [comment]
Thank you very much @squidfunk

**Comment by squidfunk** [comment]
Released as part of 8.2.3.
